CHANGELOG.md

```diff
@@ -434,54 +435,54 @@ In csritlu0, the algorithm rocsparse_itilu0_alg_sync_split_fusion has some accur
 
 rocThrust 2.18.0 for ROCm 5.7.0
 
-##### Fixed 
+##### Fixed
 
-- `lower_bound`, `upper_bound`, and `binary_search` failed to compile for certain types.
-- Fixed issue where `transform_iterator` would not compile with `__device__`-only operators.
+* `lower_bound`, `upper_bound`, and `binary_search` failed to compile for certain types.
+* Fixed issue where `transform_iterator` would not compile with `__device__`-only operators.
 
 ##### Changed
 
-- Updated `docs` directory structure to match the standard of [rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core).
-- Removed references to and workarounds for deprecated hcc
+* Updated `docs` directory structure to match the standard of [rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core).
+* Removed references to and workarounds for deprecated hcc
 
 #### rocWMMA 1.2.0
```

Comment:
Parts of the CHANGELOG.md are generated from other projects

eg: for rocWMMA https://github.com/ROCmSoftwarePlatform/rocWMMA/blob/release/rocm-rel-5.7/CHANGELOG.md